lib/charms/smtp_integrator/v0/smtp.py

```diff
@@ -280,7 +280,9 @@ def get_relation_data(self) -> Optional[SmtpRelationData]:
         relation = self.model.get_relation(self.relation_name)
         return self._get_relation_data_from_relation(relation) if relation else None
 
-    def _get_relation_data_from_relation(self, relation: ops.Relation) -> SmtpRelationData | None:
+    def _get_relation_data_from_relation(
+        self, relation: ops.Relation
+    ) -> Optional[SmtpRelationData]:
```

Comment:
Isn't Python 3.8 EOL? Perhaps we can get it removed from the relation interface repo?

Reply:
https://devguide.python.org/versions/

Reply:
Yes, but 20.04 come with Python 3.8 hasn't EoL yet, and the `charm-relation-interfaces` test is using that/

Reply:
Fair enough